pkg/daemon/ceph/osd/agent.go

```diff
@@ -629,6 +638,7 @@ func isFilestoreDevice(cfg *osdConfig) bool {
 }
 
 func isFilestoreDir(cfg *osdConfig) bool {
-	// A dir will use filestore unless explicitly requested to be bluestore (the default is blank)
-	return cfg.dir && cfg.storeConfig.StoreType != config.Bluestore
+	// All directory OSDs are filestore
+	// TODO: is this change safe?
```

Comment:
Seems fine, but similar comment to above if we should clean up the dead code at the same time